platforms/core-execution/persistent-cache/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ errorprone {
 
 dependencies {
     api(project(":base-annotations"))
-    api(project(":enterprise-logging"))
+    api(project(":build-operations"))
```

Comment:
💭 This is the reason for the PR: we want to avoid `:persistent-cache` to depend on `ProgressLogger`. This should simplify using the `PersistentCache` and by extension the build cache library by non-Gradle clients.